### DISCUSSION TITLE
RFE: "Done and next" button when navigating to an issue from github.com/notifications

### DISCUSSION THREAD

**Discussion by gauravjuvekar** [community/community#57481] [Projects and Issues]
### Select Topic Area

Product Feedback

### Select Feature Area

- [X] Issues (The feature that allows you to track ideas, feedback, tasks, or bugs for work on GitHub. Not for any issue you are having).
- [ ] Projects (For planning and tracking work on GitHub - not any open source or other type of project).

### Body

When reading an issue from github.com/notifications, add a "done and next" button at the top.

![image](https://github.com/community/community/assets/2271906/2bb668f9-2d65-4c57-81b4-2b44652b2d39)

Similar to the "done" button, instead of navigating back to github.com/notifications, this button should jump directly to the next unread notification from the notifications list.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Avivhdr** [comment]
Yes please!
